hangups/ui/__main__.py

```diff
@@ -198,11 +198,23 @@ def _on_event(self, conv_event):
 
     def _on_quit(self):
         """Handle the user quitting the application."""
-        future = asyncio.async(self._client.disconnect())
-        future.add_done_callback(lambda future: future.result())
+        asyncio.ensure_future(self._client.disconnect())
```

Comment:
Python 3.4.4 was release in 2015/12, version 3.4.6 is released. The major linux distributions (Ubuntu, Debian, Arch, Fedora, CentOS _via EPEL_ ) already have Python3.5.x in their package managers.

Raising the requirement should cause no issues.